### PR TITLE
endpoint QuotaOptions

### PR DESCRIPTION
**Pull request by jiho-kr**
Quota기능을 위한 Endpoint에 Options 추가

### CODE REVIEW

**Comment by jiho-kr** [comment]
@kson @kyseo @sham @joan

File: src/services/amqp-channel-pool-service.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export class AmqpChannelPoolService {
     if (!_.includes(this.openChannels, channel)) {
       return;
     }
-    if (reusable && this.idleChannels.length < this.options.poolSize) {
+    if (reusable && this.idleChannels.length < Number(this.options.poolSize)) {
```
Comment (heycalmdown) [inline src/services/amqp-channel-pool-service.ts]: rebase from master to pull #83 in.

File: src/controllers/endpoint-decorator.ts [hunk 2]
```diff
@@ -616,6 +622,22 @@ function pushSafe(object, arrayName, element) {
   array.push(element);
 }
 
+// endpoint에 quota를 설정한다.
+//
+// [EXAMPLE]
+// @island.quota(1, 2)
+// @island.endpoint('...')
+export function quota(limit:number, ban:number) {
```
Comment (heycalmdown) [inline src/controllers/endpoint-decorator.ts]: It would be nice to understand that `ban` is changed to `banSecs` here also.

**Comment by heycalmdown** [comment]
https://travis-ci.org/spearhead-ea/island/builds/216253260

There is lint errors. Please fix them.

**Pull request merged**